testing/src/network/mod.rs

```diff
@@ -179,22 +179,28 @@ pub fn test_config(setup: TestSetup) -> Config {
 }
 
 /// Starts a node with the specified bootnodes.
-pub async fn test_node(setup: TestSetup) -> Node<LedgerStorage> {
+pub async fn test_node(setup: TestSetup) -> Node {
     let is_miner = setup.consensus_setup.as_ref().map(|c| c.is_miner) == Some(true);
     let config = test_config(setup.clone());
-    let mut node = Node::new(config).unwrap();
-
-    if let Some(consensus_setup) = setup.consensus_setup {
-        let consensus = test_consensus(consensus_setup);
-        node.set_sync(consensus);
-    }
+    let node = match setup.consensus_setup {
+        None => Node::new(config, Arc::new(KeyValueStore::new(MemDb::new())))
```

Comment:
note: most of the storage-related tests will now work with the in-memory implementation; this needs to be documented

Reply:
not most, all